testing.go

```diff
@@ -53,6 +53,21 @@ func SkipIfDockerDesktop(t *testing.T, ctx context.Context) {
 	}
 }
 
+// SkipIfNotDockerDesktop is a utility function capable of skipping tests
+// if tests are not run using Docker Desktop.
+func SkipIfNotDockerDesktop(t *testing.T, ctx context.Context) {
```

Comment:
kudos: nice little helper.

Reply:
I was rebasing -i to remove one commit and I thought I had removed this block, used for a test including the new Docker Model Runner... but indeed, it's fine to have it